psi4/driver/procedures/proc.py

```diff
@@ -2172,8 +2172,14 @@ def run_ccenergy(name, **kwargs):
     if core.get_global_option('RUN_CCTRANSORT'):
```

Comment:
Yes, the RUN_CCTRANSORT option needs to stay in `read_options.cc`. If an option is not a valid keyword an error is thrown.